### ISSUE TITLE
load bouding boxes from previous images.

### ISSUE THREAD

**Issue by YeetMasterYeet** [Cartucho/OpenLabeling#62]
hi, i have a small suggestions to enhance open labelling, for video usage the frame boxes are basically the same, i suggest that we should add a feature that allows to load the bouding boxes from the previous frame and add a button to move the box around so that it become muc hquicker the treat data from videos.
cordially.

**Comment by fogx** [comment]
you can use a tracker to track multiple frames rapidly. Once the tracker diverges from the target just delete that frame and all following get deleted as well.

**Comment by zhangqinghao0811** [comment]
Yes, I can't agree with you any more. It's very useful for tracking a motionless object, as DaSiamRPN tracker is not stable sometimes.

**Comment by zhangqinghao0811** [comment]
> you can use a tracker to track multiple frames rapidly. Once the tracker diverges from the target just delete that frame and all following get deleted as well.

Look this [example](https://drive.google.com/open?id=1LUnmwYxddG3yP4ycDJYZMWSNqJGD4y3-), the turck is motionless, but the tracker's prediction is unstable.

**Comment by fogx** [comment]
the example is restricted, i can't see it. You might have to try some of the other trackers then. I prefer the CSRT Tracker

**Comment by zhangqinghao0811** [comment]
> the example is restricted, i can't see it. You might have to try some of the other trackers then. I prefer the CSRT Tracker

Sorry, try this [link](https://drive.google.com/file/d/1LUnmwYxddG3yP4ycDJYZMWSNqJGD4y3-/view?usp=sharing), thanks for your advise, I'll try.
